client_app/src/components/shelter/MoreDetailsBox.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const MoreDetailsBox = () => {
+
+  return (
+    <div className="modal-overlay">
+      <div className="modal-content">
```

Comment:
Add the css code for model-content. For each shift info, it should be a gray box.